types/react-fa/index.d.ts

```diff
@@ -37,7 +37,6 @@ export const Icon: ComponentClass<IconProps>;
 
 export interface IconStackProps extends SizeOverrideHTMLProps<IconStack> {
     size?: IconSize;
-    children?: IconProps[];
```

Comment:
Like I wrote, this is an example/concept that might not work out of the box. I just put it in the comment without testing it out. If you remove `children` you lose type safe check here. I cannot approve that.

Can you share the link to the changes in `@types./react` that might break it?